requirements_for_test.txt

```diff
@@ -1,12 +1,13 @@
 -r requirements.txt
-pytest==2.8.2
-pep8==1.5.7
-requests-mock==0.6.0
-mock==2.0.0
-lxml==3.4.4
+coverage==3.7.1
```

Comment:
Alphabetisation FTW. There's a flake8 plugin that enforces alphabetisation in import statements 😨  Not sure I want to lump that on people all at once though :grin:.